lib/kasket.rb

```diff
@@ -29,6 +30,7 @@ def setup(options = {})
     CONFIGURATION[:max_collection_size] = options[:max_collection_size] if options[:max_collection_size]
     CONFIGURATION[:write_through]       = options[:write_through]       if options[:write_through]
     CONFIGURATION[:default_expires_in]  = options[:default_expires_in]  if options[:default_expires_in]
+    CONFIGURATION[:dalli_allow_true_class_response] = !!options[:dalli_allow_true_class_response] if options.key?(:dalli_allow_true_class_response)
```

Comment:
isn't that the same ?
```suggestion
    CONFIGURATION[:dalli_allow_true_class_response] = !!options[:dalli_allow_true_class_response]
```